src/Symfony/Component/ErrorCatcher/ErrorRenderer/XmlErrorRenderer.php

```diff
@@ -40,37 +41,51 @@ public static function getFormat(): string
      */
     public function render(FlattenException $exception): string
     {
-        $message = $this->escapeXml($exception->getMessage());
+        $xmlDocument = new \DOMDocument('1.0', $this->charset);
+
+        $problemElement = $xmlDocument->createElement('problem');
+        $problemElement->setAttribute('xmlns', 'urn:ietf:rfc:7807');
+        $xmlDocument->appendChild($problemElement);
+
+        $titleElement = $xmlDocument->createElement('title', $this->escapeXml($exception->getTitle()));
```

Comment:
instead of manually escaping, it would be better to use https://www.php.net/manual/en/domdocument.createtextnode.php which escapes automatically

Reply:
we still need to escape attributes apparently because those do not get escaped automatically... what a nice DOM API...
so `escapeXml` could be renamed to `escapeXmlAttribute` because it also escapes `"` that do not need to be escaped inside a text node.